dev/AppLifecycle/ActivationRegistrationManager.h

```diff
@@ -6,18 +6,37 @@
 
 namespace winrt::Microsoft::ProjectReunion::implementation
 {
+    // Registration constant values.
+    static const std::wstring c_argumentPrefix = L"----";
```

Comment:
>std::wstring [](start = 17, length = 12)

See other comment re std::wstring for constants vs (maybe) std::wstring_view or PCWSTR